### ISSUE TITLE
Searchable #tags# for rooms

### ISSUE THREAD

**Issue by Allikin** [pdaddyo/soundbounce#188]
It would be nice if we could add searchable tags to a room, to make them easier to find.

**Comment by pdaddyo** [comment]
YES.  This will happen.

**Comment by pdaddyo** [comment]
Although having thought about it, it might not!! Keyword search might be better - room admins can add keywords that make their rooms appear in searches.  Basically same as tags, but not visible (so not cluttering UI). Will keep thinking about this!

**Comment by Allikin** [comment]
I agree, I wasn't really on about visible keywords anyway, but more a keyword field like you say.
